src/controllers/v1.controller.ts

```diff
@@ -173,7 +175,17 @@ export class V1Controller {
 
       // Fetch applications contained in this Load Balancer. Verify they exist and choose
       // one randomly for the relay. First check RPC ID to see if this client should be stuck to an app and node.
-      const { preferredApplicationID, preferredNodeAddress, rpcID } = await this.checkClientStickiness(rawData)
+
+      const { stickiness = false, stickinessDuration = DEFAULT_STICKINESS_DURATION } = loadBalancer
+
+      const stickinessData = stickiness
+        ? await this.checkClientStickiness(rawData)
+        : {
+            preferredApplicationID: '',
+            preferredNodeAddress: '',
+            rpcID: 0,
+          }
```

Comment:
We could probably lift this up to the top of the file as a constant, and name it `DEFAULT_STICKINESS_PARAMS` or something similar.